libs/barista-components/alert/BUILD.bazel

```
@@ -16,7 +39,54 @@ filegroup(
     ]),
 )
 
+sass_library(
+    name = "scss_lib",
```

I would rename to `theme` as it is more descriptive
```suggestion
    name = "theme",
```